Terminal.Gui/Application/Application.Keyboard.cs

```diff
@@ -97,17 +97,11 @@ public static bool RaiseKeyDownEvent (Key key)
             }
             else
             {
-                // BUGBUG: this seems unneeded.
```

Comment:
Are you 100% sure this won't break anything? 

I am not, which is why I have not previously cleaned it up.

Reply:
I thinking it's a duplicated running always with the same returning result and thus I think it's unneeded. I didn't experienced any situation when the second call was different from prior. Also no unit test fail which I think it's secure to remove it.